### ISSUE TITLE
Try to use dcn, but find the effect is not good

### ISSUE THREAD

**Issue by kaikaizhu** [open-mmlab/mmdetection#1507]
I try to use dcn and modify the config file cascade_rcnn_r101_fpn_1x.py:

model = dict(
    type='CascadeRCNN',
    num_stages=3,
    pretrained='torchvision://resnet101',
    backbone=dict(
        type='ResNet',
        depth=101,
        num_stages=4,
        out_indices=(0, 1, 2, 3),
        frozen_stages=1,
        style='pytorch',
        dcn=dict(
            modulated=False,
            deformable_groups=1,
            fallback_on_stride=False),
        stage_with_dcn=(False, True, True, True)
    ),

I just added the following paragraph,then the effect becomes very bad.:

        dcn=dict(
            modulated=False,
            deformable_groups=1,
            fallback_on_stride=False),
        stage_with_dcn=(False, True, True, True)

Can you tell me what the problem is?

**Comment by kaikaizhu** [comment]
I change the backbone, however, I used the resnet101-5d3b4d8f.pth as the pretrain weight. I don't know if this is the right thing to do. Can anyboby help me?
@hellock

**Comment by hznuhisejeffrey** [comment]
Hello，I have the same problem as you，have you solved the problem？

**Comment by ZwwWayne** [comment]
When you say the effect becomes very bad, what do you mean? You mean the loss blow up? Or the mAP becomes lower?
The description is kind of vague, thus we are not able to help you if you do not provide more information (e.g., training log and full configs, etc.).
